### DISCUSSION TITLE
all-day event spanning multiple days in month view

### DISCUSSION THREAD

**Discussion by phuanggh** [kvyatkovskys/KVKCalendar#314] [Ideas]
In the month view, can it displays all-day event spanning multiple days without cutting the event bar like the native calendar?
![image](https://user-images.githubusercontent.com/51040483/231045805-e3ae1475-ac4e-4a9b-b56a-2ace743955b5.png)

**Comment by kvyatkovskys** [comment]
Hello! 
Right now no... I plan to rewrite on the library SwiftUI and after I'll try to implement this ability.
